h2o-py/tests/testdir_algos/gbm/pyunit_4007_decision_tree_language_rules_printing.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import print_function
+from builtins import range
+import sys
+from h2o.tree import H2OTree
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils, math
+from h2o.estimators.gbm import H2OGradientBoostingEstimator
+
+
+def decision_tree_language_rules_printing():
+    print(" ----- numerical case: -----")
+    df = h2o.import_file(path=pyunit_utils.locate("smalldata/logreg/prostate.csv"))
+    df.describe()
+    train = df.drop("ID")
+    vol = train['VOL']
+    vol[vol == 0] = None
+    gle = train['GLEASON']
+    gle[gle == 0] = None
+    train['CAPSULE'] = train['CAPSULE'].asfactor()
+    train.describe()
+    my_gbm = H2OGradientBoostingEstimator(ntrees=50, learn_rate=0.1, distribution="bernoulli", max_depth = 2, seed = 12345)
+    my_gbm.train(x=list(range(1, train.ncol)),
+                 y="CAPSULE",
+                 training_frame=train,
+                 validation_frame=train)
+    first_tree = H2OTree(model = my_gbm, tree_number = 0, tree_class = None)
+
+    print(" -- Tree predictions: -- ")
+    print(first_tree.predictions)
+    
+    print(" -- Language tree representation: -- ")
+    assert first_tree.language_tree_representation is not None
+    print(first_tree.language_tree_representation)
+    text_file = open("pyunit_4007_language_tree_representation_numerical_case.txt", "r")
```

Comment:
This expects the working dir to be in the test dir, but I think it is not, pyunit_utils.locate should resolve that

Reply:
fixed